DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py

```diff
@@ -294,55 +294,55 @@
 if (process.runType.getRunType() == process.runType.pp_run or process.runType.getRunType() == process.runType.pp_run_stage1):
     #event selection for pp collisions
     if ((process.runType.getRunType() == process.runType.pp_run) and live):
-        process.source.SelectEvents = cms.untracked.vstring(
+        process.source.SelectEvents = [
             'HLT_L1*',
             'HLT_Jet*',
             'HLT_Physics*',
             'HLT_ZeroBias*',
             'HLT_PAL1*',
             'HLT_PAZeroBias*',
             'HLT_PAAK*'
-            )
+            ]
 
     # Source and Client config for pp collisions
 
-    process.SiStripMonitorDigi.UseDCSFiltering = cms.bool(False)
-    process.SiStripMonitorClusterReal.UseDCSFiltering = cms.bool(False)
+    process.SiStripMonitorDigi.UseDCSFiltering = False
+    process.SiStripMonitorClusterReal.UseDCSFiltering = False
 
     process.MonitorTrackResiduals_gentk.Tracks                 = 'initialStepTracksPreSplitting'
     process.MonitorTrackResiduals_gentk.trajectoryInput        = 'initialStepTracksPreSplitting'
-    process.MonitorTrackResiduals_gentk.TrackProducer          = cms.string('initialStepTracksPreSplitting')
-    process.TrackMon_gentk.TrackProducer    = cms.InputTag('initialStepTracksPreSplitting')
-    process.TrackMon_gentk.allTrackProducer = cms.InputTag('initialStepTracksPreSplitting')
+#    process.MonitorTrackResiduals_gentk.TrackProducer          = 'initialStepTracksPreSplitting' ##DOES NOT EXIST ON DQM/TrackerMonitorTrack/src/MonitorTrackResiduals.cc
```

Comment:
@mtosi @mmusich if you could please confirm? Thanks!

Reply:
the change *per se* looks OK (please remove the commented code as well), though the module is not cloned here while in the PR description there's written:
> ... "hidden" non-existing/wrong parameters or not actually declared in the original cloned module, have been identified in the process